css/_includes/_banner.scss

```diff
@@ -0,0 +1,200 @@
+.banner {
+  @include u-w100;
+}
+
+.banner--hero,
+.banner--interior-hero {
+  @include flex-container-centered;
+  @include t-center;
+  min-height: 420*$em;
+  height: 40vw;
+  max-height: 600*$em;
+  background: $banner-hero url("../assets/banner.jpg") center center no-repeat;
+  background-size: cover;
+}
+
+.banner--hero__content {
+  padding-top: 36*$em;
+  @include fn-filter(drop-shadow(0 0 50*$em rgba(0,0,0,.3)));
+}
+
+.banner--hero,
+.banner--interior-hero {
+  .banner--hero__content,
+  .constrained.constrained--sm {
+    padding-left: $nav-padding;
+    padding-right: $nav-padding;
+  }
+}
+
+.banner--hero__logo,
+.banner--hero__content h3 {
+  @include u-mp0;
+  @include font-size-reset;
+  cursor: default;
+}
+
+.banner--hero__logo svg {
+  fill: $white;
+  width: 265*$em;
+  height: 46*$em;
+}
+
+.banner--hero__content h3 {
+  margin: 30*$em 0;
+
+  span {
+    @include t-thin;
+    @include t-crisp;
+    font-size: 20*$em;
+    color: $white;
+  }
+}
+
+.banner--hero__cta-container {
+  @include flex-container-centered;
+  padding-top: 36*$em;
+
+  .btn {
+    @include u-w100;
+    max-width: 224*$em;
+  }
+
+  .btn + .btn {
+    margin-left: 36*$em;
+  }
+}
+
+.banner__ai2-logo {
+  opacity: .8;
+
+  svg {
+    font-size: 7*$em;
+    width: 52*$em;
+    height: 39*$em;
+    margin-left: 4*$em;
+    margin-bottom: -1*$em;
+  }
+
+  &:hover {
+    opacity: 1;
+  }
+}
+
+.banner--interior-hero {
+  @include t-white;
+  min-height: 274*$em;
+  height: auto;
+  background-color: $muted-gray-blue;
+  background-image: url("../assets/banner-blurred.jpg");
+  position: relative;
+
+  &:before {
+    @include u-pe100;
+    background: black;
+    z-index: -1;
+  }
+}
+
+.banner--interior-hero__content {
+  padding: 20*$em;
+  padding-bottom: 0;
+  max-width: 620*$em;
+}
+
+.banner--padded .constrained {
+  padding: 76*$em $nav-padding * 2;
+
+  h2,
+  p {
+    @include u-mp0;
+  }
+
+  h2 + p {
+    margin-top: 32*$em;
+  }
+
+  p {
+
+  }
+}
+
+@media screen and (max-width: 960px) {
+  .banner--hero {
+    .banner--hero__content {
+      padding: $nav-padding * 2 $nav-padding;
+
+      h3 {
+        margin-bottom: 0;
+      }
+
+      .banner--hero__cta-container .btn + .btn {
+        margin-left: $nav-padding;
+      }
+    }
+
+    h1.banner--hero__logo {
+      font-size: 12*$em;
+    }
+  }
+
+  .banner--hero,
+  .banner--interior-hero {
+    min-height: 200*$em;
+    height: auto;
+
+    .constrained.constrained--sm {
+      padding-top: 16*$em;
+      padding-bottom: 20*$em;
+
+      h1 {
+        margin-top: 0;
+      }
+    }
+  }
+}
+
+/* Post-launch edits */
```

Comment:
@schmmd heads up, these are changes that we made when we added the AI2 bar at the bottom of the hero banner.

Reply:
(Copied over from the manually-edited compiled CSS file)